README.md

```diff
@@ -39,8 +51,14 @@ make
 Finally, build and run the example app:
 
 ```
-cd examples/hello
-pipenv install
-spin build
+cd examples/hello_world
+../../target/release/spin-python app -o app.wasm
```

Comment:
_Please_ explain that `spin-python` and `py2wasm` are the same thing.  Without that key piece of info, it feels like there are two completely different things going on here, rather than the same commands being run via Spin vs standalone.

Reply:
Makes sense. Added it.

Reply:
Side question: why the different names?

Reply:
`spin-python` was the original name and has remained since. when using the binary in a standalone context it is more descriptive with the prefix of `spin`. It is the same way as the JS SDK plugin as well. I do not have strong feelings about it. Happy to rename the binary

cc: @dicej